src/seedu/addressbook/data/person/Address.java

```diff
@@ -22,10 +27,19 @@
      */
     public Address(String address, boolean isPrivate) throws IllegalValueException {
         this.isPrivate = isPrivate;
-        if (!isValidAddress(address)) {
+        
+        String[] partsOfAddress = address.split(", ");
```

Comment:
I noticed that address is split by ", ". This will cause addresses entered in the form of "123,Clementi Ave 3,#12-34,231534" to have problems. Do you think we should accept that type of input for addresses? A suggestion to split by "," will have more leeway for inputs.